Minor changes to docs

rjanja opened the pull request.

Commit:
Minor changes to docs

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
-
+* Document how LEDs can be specified or individually addressed in `blink_leds` command
+* Document OTA update of Wi-Fi module
+  - Firmware link no longer works
```
Looks like it links to `https://my.st.com/content/my_st_com/en/products/embedded-software/wireless-connectivity-software/wireless-connectivity-software/stsw-wifi001.html` instead of `https://my.st.com/content/my_st_com/en/products/embedded-software/wireless-connectivity-software/stsw-wifi001.html` (one `wireless-connectivity-software` too many). Will have to update `firstset.html`.

rjanja replied:
Ahh, great! I'm downloading the Wi-Fi module firmware now 👍

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
+  - Firmware link no longer works
+  - Is this process safe or can it brick the device?
+  - Is it essential? 
+  - What does it improve?
```
Updating the Wi-Fi modules firmware is very safe, I have done it so many times I've lost count while testing and ensuring the process works smoothly, both in error / success situations 😄. Also, I've performed it on 4 different units.

I don't have a changelog at hand, but you newer versions of the firmware fixes a lot of bugs. The latest (official) version is `3.5` but I've been beta-testing a newer version that further fixes Wi-Fi connectivity issues and bugs in the socket server that I've reported.

If you download the `3.5` firmware you will be able to see a changelog between `3.4` and `3.5`.

I'll put some thought into documenting this better 🙂.

rjanja replied:
Excellent! I'll proceed w/ updating my bttn here (and removing these questions from README)

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
 
 Installing this firmware will most likely void your warranty.
 
+> Herein, **bt.tn** refers to The Button Corporation Ltd., and **bttn** refers to a consumer device manufactured by said company.
```
Nice, thanks for introducing this consistency 👍!

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
+### Boot bttn in DFU mode
+
+1. Open the bttn by removing the three T10 torx screws ![torx](resources/torx.png "Torx head") at the bottom.
+2. Connect the [Boot pins](./resources/bttn-boot-pins.png) on the board (note: only the Boot pins, not the FWSel ones). An easy way to do this is with [a paper clip and a bit of electrical tape](./resources/bttn-boot-pins-shorting.png) twice-folded so as to insulate the paper clip so that it does not touch the FWSel pins or any other contacts on the board.
```
Lovely ❤️!

mafredri replied:
Here `./resources/bttn-boot-pins-shorting.png` should be `./resources/bttn-boot-pins-shorting.jpg`.

rjanja replied:
Good catch!

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
+
+### Built-in socket server
+
+There are several functions available via the socket server. Note that URLs must be encoded to AT format before being passed to the socket server. Refer to [at.js](public/at.js) for more information.
```
This should be `[at.js](public/openbttn/at.js)`.

rjanja replied:
👍

mafredri commented on README.md:
```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
+  - Firmware link no longer works
+  - Is this process safe or can it brick the device?
+  - Is it essential? 
+  - What does it improve?
```
Here `./resources/bttn-boot-pins-shorting.png` should be `./resources/bttn-boot-pins-shorting.jpg`.

mafredri reviewed (approved):
Wow, this is a big improvement to the readme, thanks @rjanja! I really appreciate it.

I noticed a few minor issues, but no biggie if you don't have to fix them, I can do it after merging as well 😄.

Commit:
Fix links and typos

rjanja commented:
Ok, I think I've addressed your suggestions. I added the Wi-Fi module firmware link to the README, because there's a short `goo.gl` link in `firstset.html` that I didn't want to mess with in a docs PR. Thanks!

mafredri commented:
Great, thanks again! Also, let me know how the Wi-Fi firmware upgrade went 😄!

rjanja commented:
I'm looking into that now. Trying to figure out if I need to recompile and re-flash the bttn while having the Wi-Fi image located in the `ota` folder, or whether I need to run a webserver to serve it. I'm not sure what should be located at `192.168.1.2:8774/ota/`
